projects/schematics/src/migrations/4_0/methods-and-properties-deprecations/data/http-error-model.migration.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  ERROR,
+  HTTP_ERROR_MODEL,
+  SPARTACUS_CORE,
+  TODO_SPARTACUS,
+} from '../../../../shared/constants';
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const HTTP_ERROR_MODEL_MIGRATION: MethodPropertyDeprecation[] = [
+  {
+    class: HTTP_ERROR_MODEL,
+    importPath: SPARTACUS_CORE,
+    deprecatedNode: ERROR,
+    comment: `// ${TODO_SPARTACUS} Property '${ERROR}' was removed from interface`,
+  },
+];
```

Comment:
We don't need migration for interface changes. In ticket I was referring only to provide migration for `UserSignUp`. Contact with @Matejk00 if he covers this model in his PR